spec/controllers/api/v1/leases_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::LeasesController, type: :controller do
+  login_admin
+
+  application_with_lease = 'a0o210000027w7EAAQ' # All Preferences
+  test_lease_id = 'a1321000000qBsrAAE' # Lease id for application_with_lease
+  application_without_lease = 'a0o0P00000IvkTqQAJ' # Tristan Maleah - Lease needs to be deleted if VCR re-taped
+
+  describe '#create' do
+    it 'updates existing lease if application has lease' do
+      VCR.use_cassette('api/v1/leases/create/app-with-lease') do
+        params = {
+          application_id: application_with_lease,
+          lease: {
+            'monthly_parking_rent': 100,
+          },
+        }
+        post :create, params: params
+      end
+      expect(response).to have_http_status(:success)
+      json = JSON.parse(response.body)
+      expect(json['lease']).to eq(true) # update returns true
+    end
+    it 'creates a new lease if application doesn\'t have lease' do
+      VCR.use_cassette('api/v1/leases/create/app-without-lease') do
+        params = {
+          application_id: application_without_lease,
+          lease: {
+            'monthly_parking_rent': 100,
+          },
+        }
+        post :create, params: params
+      end
+      expect(response).to have_http_status(:success)
+      json = JSON.parse(response.body)
+      expect(json['lease'].size).to eq(18) # create returns lease id
+    end
+  end
+
+  describe '#update' do
+    it 'updates a lease successfuly' do
```

Comment:
@akegan tiny typo "successfuly" -> "successfully"
```suggestion
    it 'updates a lease successfully' do
```